### ISSUE TITLE
Could not open input file error on format

### ISSUE THREAD

**Issue by gab-globusgroup** [kokororin/vscode-phpfmt#104]
Hello and thank you for this great plugin!

Since the last update, when I format php files it looks like it prepends my current working folder to the path to the formatter, generating this error:

```
Could not open input file: /Users/gabriele/myprojectfolder/app/Http/Controllers//Users/gabriele/.vscode/extensions/kokororin.vscode-phpfmt-1.1.1/node_modules/use-phpfmt/fmt-external.stub.php
```

I didn't change anything in the configuration and I tried to restart vscode/reinstall the plugin with no success.

This are my settings for phpfmt:
```json
"phpfmt.enable_auto_align": false,
"phpfmt.php_bin": "/opt/homebrew/opt/php@8.0/bin/php",
"phpfmt.passes": [
    "SpaceAfterExclamationMark",
],
"phpfmt.exclude": [
    "PSR2ModifierVisibilityStaticOrder"
],
"[php]": {
    "editor.defaultFormatter": "kokororin.vscode-phpfmt"
},
```

Am I missing some new setting? Any help debugging this would be much appreciated, thank you!

**Comment by kokororin** [comment]
Can you open phpfmt output to see what prints?

**Comment by kokororin** [comment]
Fixed in 1.1.2
